src/pages/study-rooms/components/study-room-list-item.tsx

```diff
@@ -1,23 +1,29 @@
-interface StudyRoomProps {
-  id: number;
-  title: string;
-  count: string;
-}
-const StudyRoomListItem = ({ id, title, count }: StudyRoomProps) => {
+import { Chip, ListItem, ListItemButton, ListItemText } from '@mui/material';
+import { StudyRoom } from '@/types/study-rooms.types';
+import { studyRoomStep } from '@/constant/study-room';
+
+type StudyRoomProps = StudyRoom;
+
+const MAX_PARTICIPANT_COUNT = 8;
+
+const StudyRoomListItem = ({ name, step, participantCount }: StudyRoomProps) => {
```

Comment:
이렇게 되는게 네이밍이 일관적일 것 같네요!
```suggestion
const StudyRoomListItem = ({ name, step, participantCount }: StudyRoomListItemProps) => {
```